typescript/sdk/src/deploy/HyperlaneAppChecker.ts

```diff
@@ -82,6 +83,10 @@ export abstract class HyperlaneAppChecker<
   }
 
   addViolation(violation: CheckerViolation): void {
+    if (violation.type === ViolationType.BytecodeMismatch) {
+      rootLogger.warn({ violation }, `Found bytecode mismatch. Ignoring...`);
```

Comment:
should we do something more here?

Reply:
i think we'll have to fix it properly with something like this  https://discord.com/channels/935678348330434570/1296158577765126145/1296165727996350618